semcore/date-picker/src/components/Calendar.jsx

```diff
@@ -261,13 +325,17 @@ class CalendarDaysRoot extends CalendarAbstract {
     const SCalendar = Root;
     const SGridDays = 'div';
     const { Children, styles, locale } = this.asProps;
+    const description = this.describeValue();
 
     return sstyled(styles)(
-      <SCalendar render={Box} aria-hidden='true'>
+      <SCalendar render={Box}>
         <CalendarWeekDays locale={locale} />
         <SGridDays onMouseLeave={this.handleMouseLeave}>
           <Children />
         </SGridDays>
+        <span aria-live='polite'>
+          <ScreenReaderOnly>{description}</ScreenReaderOnly>
```

Comment:
maybe we should add `aria-live` to the `ScreenReaderOnly` as prop?

Reply:
No, `ScreenReaderOnly` isn't based on Root magic and doesn't accepts DOM props

Reply:
ok, but maybe we should change order of compoents and put select with `aria-live` in `ScreenReaderOnly`?

Reply:
Sure, that's going to be more correct